packages/lime-smart-wifi/Makefile

```diff
@@ -1,6 +1,7 @@
 LIME_DESCRIPTION:=Automatically configure wifi mesh and ap based on existing radios
 LIME_MAINTAINER:=Paul Spooren <paul@spooren.de>
 LIME_TITLE:=Configure wifi mesh and ap hardware based
+LIME_DEPENDS:=+lime-system
```

Comment:
Are you really really sure `lime-system` is needed?

Reply:
https://github.com/libremesh/lime-packages/issues/766#issuecomment-702940526
and
https://github.com/libremesh/lime-packages/blob/a8afa88772ab549089adfe28a247a28d08a5f46c/packages/lime-smart-wifi/files/usr/bin/lime-smart-wifi.lua#L3-L5